src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -399,6 +408,7 @@ export default class FilingHistoryList extends Mixins(
   private currentFiling: HistoryItemIF = null
   private loadingOneIndex = -1
   private isBusy = false
+  private correctionFiling: HistoryItemIF = null
```

Comment:
Any chance you could reuse `currentFiling` (line 408) instead of declaring a new variable?

I am OK with a new variable except that it's named exactly the same as the correction filing on line 689, which may lead to confusion/errors.